bumble/host.py

```diff
@@ -487,8 +488,8 @@ def supported_commands(self):
 
         return commands
 
-    def supports_le_feature(self, feature):
-        return (self.local_le_features & (1 << feature)) != 0
+    def supports_le_feature(self, feature: LeFeatureMask) -> bool:
```

Comment:
Not necessarily a problem, but this is a different method compared to before: instead of asking for just one feature, this asks for an entire set of features.
We have to options here:
1/ Either we keep the old semantics, and just ask for one feature at a time, in which case we should have an `LeFeature` simple IntEnum (we could also have an `LeFeatureMask` as an IntFlags if we wanted an actual type for it), or
2/ rename this method to `supports_le_features`, taking a whole mask/set, but in that case it should return `self.local_le_features & features == features` (i.e "all features" are supported, not just "one of the features" is supported, which isn't terribly useful because you don't know which one)

Reply:
Agree. But I think it's better to define flags as they are, instead of their log position. So 2/ is the one I preffered.